app/helpers/metadata_helper.rb

```diff
@@ -107,7 +110,7 @@ def validate_metadata_csv_for_samples(samples, metadata)
         field = metadata["headers"][row_index]
 
         # Ignore invalid columns.
-        if field != "sample_name" && MetadataField.find_by(name: field)
+        unless field == "sample_name" || MetadataField.find_by(name: field) || MetadataField.find_by(display_name: field)
```

Comment:
Should this condition be `unless field == "sample_name" || (!MetadataField.find_by(name: field) && !MetadataField.find_by(display_name: field))`?

Reply:
thanksss